terraform/ec2/win/main.tf

```diff
@@ -170,7 +173,7 @@ resource "null_resource" "integration_test_run_validator" {
   count = length(regexall("/feature/windows", var.test_dir)) > 0 ? 1 : 0
   depends_on = [
     null_resource.integration_test_setup,
-    null_resource.integration_test_reboot,
+    null_resource.integration_test_wait,
```

Comment:
null_resource.integration_test_wait does this step add any value for feature/windows test?

Reply:
no, but `_wait` resource doesn't even run with feature test. It is there just for more clear dependency chain.